app/forms/undo_withdrawal_form.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class UndoWithdrawalForm
+  include ActiveModel::Model
+
+  attr_accessor :comment, :ticket, :trainee, :session
+
+  validates :comment, presence: true
+  validates :ticket, format: URI::DEFAULT_PARSER.make_regexp(%w[http https]), allow_blank: true
+
+  def initialize(trainee: nil, comment: nil, ticket: nil, session: nil)
```

Comment:
It looks like `trainee` and `session` are minimal requirements.

```suggestion
  def initialize(trainee:, session:, comment: nil, ticket: nil)
```